.github/workflows/check-advisories.yml

```diff
@@ -19,12 +19,12 @@ jobs:
           name: ${{ inputs.artifact-name }}
           path: ~/.local/bin
       - run: chmod +x ~/.local/bin/hsec-tools
-      - name: run checks
+      - name: Run advisories checks
         run: |
           cd source
           RESULT=0
-          find advisories EXAMPLE_ADVISORY.md -type f -name "*.md" | while read FILE ; do
+          while read FILE ; do
             echo -n "$FILE: "
             hsec-tools check < "$FILE" || RESULT=1
-          done
+          done < <(find advisories EXAMPLE_ADVISORY.md -type f -name "*.md")
```

Comment:
I am curious why the old form did not result in exit status = 1 with bad advisory data, but the new form acts correctly.  Shell is weird.  Do you know what is the reason?

Reply:
Here's the answer: https://stackoverflow.com/questions/16854280/a-variable-modified-inside-a-while-loop-is-not-remembered/16854326#16854326

Quite the "gotcha".  Thanks for detecting and fixing this bug!